rules/falco_rules.yaml

```diff
@@ -3397,6 +3397,18 @@
   output: "File execution detected from /dev/shm (proc.cmdline=%proc.cmdline connection=%fd.name user.name=%user.name user.loginuid=%user.loginuid container.id=%container.id evt.type=%evt.type evt.res=%evt.res proc.pid=%proc.pid proc.cwd=%proc.cwd proc.ppid=%proc.ppid proc.pcmdline=%proc.pcmdline proc.sid=%proc.sid proc.exepath=%proc.exepath user.uid=%user.uid user.loginname=%user.loginname group.gid=%group.gid group.name=%group.name container.name=%container.name image=%container.image.repository)"
   priority: WARNING
   tags: [host, container, mitre_execution, mitre_defense_evasion, T1036.005, T1059.004]
+ 
+- list: open_appln_windows
+  items: ["xwininfo", "wmctrl", "xprop", "xlsclients", "xdotool", "xrestop", "xwininfo", "xdpyinfo", "glxinfo", "xrandr", "xset", "xdriinfo", "xhost", "xev", "xdotool", "xsetroot"]
+   
+- rule: Get Information About Open Application Windows
+  desc: Detects attempts to get information about open application windows
+  condition: >
+    spawned_process and container and
```

Comment:
is this rule designed to be just in container or on Host as well? I suggest to modify the Rule Name accordingly. In the condition you added in container so I suggest to add to change the name in " Launch Application Windows in Container". In case you want to apply this rule on Host as well I suggest to remove the container from the condition.

Reply:
The rule is designed to identify the activity of getting information about open application windows in a container as well as in Linux host. So I will remove the container from the rule conditions.Pushing the changes right away.